src/clientSideScene/segments.ts

```diff
@@ -1701,6 +1703,35 @@ function createLengthIndicator({
       console.error('Unable to dimension segment when clicking the label.')
       return
     }
+
+    if (engineCommandManager.kclManager !== null) {
+      if (
+        selection.graphSelections &&
+        selection.graphSelections[0] &&
+        selection.graphSelections[0].codeRef
+      ) {
+        const graphSelection = selection.graphSelections[0] as Selection
+        const node = getNodeFromPath<Expr>(
+          engineCommandManager.kclManager.ast,
+          graphSelection.codeRef.pathToNode,
+          ['CallExpressionKw']
+        )
+        if (!err(node)) {
+          const hasEndAbsolute =
+            node.node.type === 'CallExpressionKw' &&
+            node.node.arguments[0] &&
+            node.node.arguments[0].label.name === 'endAbsolute'
+          if (hasEndAbsolute) {
+            // Won't be able to constrain that length
+            toast.error(
+              'Unable to constraint the length of a segment with endAbsolute'
```

Comment:
```suggestion
              'Unable to constrain the length of a segment with endAbsolute'
```

Just a little typo imho